project/Dependencies.scala

```diff
@@ -47,7 +47,7 @@ object Dependencies {
   lazy val squeryl                = "org.squeryl"                % "squeryl"            % "0.9.5-7"   cross CVMappingAll
   lazy val slf4j_api              = "org.slf4j"                  % "slf4j-api"          % slf4jVersion
   lazy val scala_xml              = "org.scala-lang.modules"     %% "scala-xml"         % "1.0.6"
-  lazy val rhino                  = "org.mozilla"                % "rhino"              % "1.7.7.1"
+  lazy val rhino                  = "org.mozilla"                % "rhino"              % "1.7.10"
```

Comment:
I know this was a million years ago but any idea for the reason behind it?

No biggie -- I'm just currently in the flail-in-the-dark phase of bumping us to Lift 3.4.3 and am going error by eror 😅 .

Reply:
I think most of this PR was just getting us up to date with regard to our outdated dependencies.

Reply:
iiiiinterestingly enough:

we upgraded to lift 3.3 and our tests started failing. adding `exclude("org.mozilla", "rhino"),` to our lift dependencies got 'em to pass O___o.

not sure if we're just postponing the inevitable but i found it noteworthy.

Reply:
that's what got me asking about _why_ this updated dependency -- we were only successful after we reverted it.

Reply:
It's possible that you have other dependencies that rely on there being an older version of rhino present and Lift bringing in a newer version transitively causes issues for you. I'd recommend taking a look at what other dependencies might ask for Rhino (or look for any direct dependencies on it in your project) to try and determine whether or not that's what is happening. If you use Rhino directly, there's a good chance you'll have code to update.

Reply:
There is direct usage in that codebase fwiw 😉

Reply:
> It's possible that you have other dependencies that rely on there being an older version of rhino present

ooh yeah. I think that's why it the `exclude` worked, yah?

> and Lift bringing in a newer version transitively causes issues for you.

Reply:
> It's possible that you have other dependencies that rely on there being an older version of rhino present and Lift bringing in a newer version transitively causes issues for you.

ooh yeah. I think that's why it the `exclude` worked, yah?

I suppose I expected something Rhino-y in this PR. That is, instead of just bumping a version, maybe something like `rhino.oldFunction -> rhino.newFunction`. On the other hand, version bumps to stay current -- with no code changes -- are also legit.

> There is direct usage in that codebase fwiw 😉

'preciate it :D Suppose I might have to Actually Dig In:tm: & learn myself some Clojure 😎.

Reply:
Caveat -- I say "Clojure" 'cause I think that's how we're using it. Prolly shoulda [read the README first](https://github.com/mozilla/rhino) 😅 .